src/main.js

```diff
@@ -3,9 +3,9 @@ import Api from './Entities/Api';
 
 export default class JWTAuthApi {
   constructor(config, refreshUrl, refreshTTL, options) {
-    const { storageNamespace } = { ...options };
+    const { storageType, storageNamespace } = { ...options };
```

Comment:
nieśmiało wnioskuję, że `options` przychodzi jako obiekt, więc niezbyt widzę sens w spreadzie i owijaniu tego ponownie w obiekt

Reply:
To nie moje, ale nie poprawiałem, bo nie chciałem zepsuć. Poprawię